src/beast/beast/hash/hash_append.h

```diff
@@ -173,454 +213,256 @@ static_assert (is_contiguously_hashable <
 
 template <class Hasher, class T>
 inline
```

Comment:
You didn't change it here, but the block comment above this function (lines 187-209) could really use some updating.